libs/wire-api-federation/src/Wire/API/Federation/Domain.hs

```diff
@@ -58,3 +59,6 @@ instance
 
 originDomainHeaderName :: IsString a => a
 originDomainHeaderName = fromString $ symbolVal (Proxy @OriginDomainHeaderName)
+
+instance (HasOpenApi api) => HasOpenApi (OriginDomainHeader :> api) where
```

Comment:
i'm not sure, but shouldn't `OriginDomainHeader` also be documented somehow?  If in doubt, add something to the description?